attendance/views.py

```diff
@@ -1,11 +1,26 @@
-from django.http import HttpResponse, JsonResponse
+from django.http import HttpResponse
+from django.http import JsonResponse
+from django.http import HttpResponseRedirect
```

Comment:
ok by me i guess. could get pretty verbose if importing lots of individual things. my ide highlights unused imports and i'm not clear how it helps flake8 but i guess for diff outputs it would add clarity